src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -6362,6 +6362,13 @@ End Module
         <WorkItem(5664, "https://github.com/dotnet/roslyn/issues/5664")>
         <ConditionalFact(GetType(IsEnglishLocal))>
         Public Sub Bug15538()
+            ' The icacls command fails on our Helix machines And it appears to be related to the use of the $ in 
+            ' the username. 
```

Comment:
Should this filter on the $ in the username then, and possibly with a ConditionalFact? I could totally imagine this breaking VSTS runs or anything else too, and we wouldn't be aware the test was being skipped.

Reply:
@jasonmalinowski I first tried the `$` but this actually triggers on ever more machines than the current check and there are machines where `$` in the name works fine. This was the minimal check I could find given that there is almost no information on what this error is or why we are hitting it. 

Honestly I even considered just straight up deleting this test. It's not providing a ton of value, it's not portable and it's apparently far less reliable than we thought. 

This isn't going to be sitting idle. I'm engaged in either fixing this or deleting the test. I need to disable it though for now in this case so that the VSTS changes can flow more easily through our branches.

Reply:
Is there a bug tracking the fix?

Reply:
Yes. Look immediately below this comment. :) 

I linked in both code changes